File: VB/WpfApplication1/PdfSelectionChangedEventArgs.vb
Imports DevExpress.Pdf
Imports System

Namespace WpfApplication1

    Public Delegate Sub PdfSelectionChangedEventHandler(ByVal sender As Object, ByVal args As PdfSelectionChangedEventArgs)

    Public Class PdfSelectionChangedEventArgs
        Inherits EventArgs

        Private _StartPosition As PdfDocumentPosition, _EndPosition As PdfDocumentPosition

        Public Property StartPosition As PdfDocumentPosition
            Get
                Return _StartPosition
            End Get

            Private Set(ByVal value As PdfDocumentPosition)
                _StartPosition = value
            End Set
        End Property

        Public Property EndPosition As PdfDocumentPosition
            Get
                Return _EndPosition
            End Get

            Private Set(ByVal value As PdfDocumentPosition)
                _EndPosition = value
            End Set
        End Property

        Public Sub New(ByVal startPosition As PdfDocumentPosition, ByVal endPosition As PdfDocumentPosition)
            Me.StartPosition = startPosition
            Me.EndPosition = endPosition
        End Sub
    End Class
End Namespace

File: CS/WpfApplication1/PdfSelectionChangedEventArgs.cs
using DevExpress.Pdf;
using System;
using System.Linq;

namespace WpfApplication1
{
    public delegate void PdfSelectionChangedEventHandler(object sender, PdfSelectionChangedEventArgs args);
    public class PdfSelectionChangedEventArgs : EventArgs
    {
        public PdfDocumentPosition StartPosition { get; private set; }
        public PdfDocumentPosition EndPosition { get; private set; }

        public PdfSelectionChangedEventArgs(PdfDocumentPosition startPosition, PdfDocumentPosition endPosition)
        {
            StartPosition = startPosition;
            EndPosition = endPosition;
        }
    }
}
